src/redux/books/books.js

```diff
@@ -0,0 +1,26 @@
+const ADD = 'ADD';
+const REMOVE = 'REMOVE';
+
```

Comment:
- [ ] In keeping with the ducks' pattern requirement we should format these action types to something like this:

  ```
  const ADD = 'bookstore/books/ADD';
  const REMOVE = 'bookstore/books/REMOVE';
  ```